zng/ztests/enum-err.yaml

```diff
@@ -0,0 +1,9 @@
+zql: "put s=e:string"
+
+input: |
+  #0:record[e:enum[int32,foo:[1],bar:[2],baz:[4]]]
+  0:[100;]
+
+output: |
+  #0:record[e:enum[int32,foo:[1],bar:[2],baz:[4]],s:error]
```

Comment:
I know we've talked about this on another PR, but seeing this confirms my impression that it will be hard to work with. I guess we can let it play out and get some mileage on it (this doesn' tneed to hold up this PR as it's not zng).

Reply:
Agreed.  Let's get this in the commit history in this general way then we can change or delete.